### PR TITLE
Android: Fix white screen of death.

### PR DESCRIPTION
**Pull request by strseb**

### CODE REVIEW

File: nebula/ui/components/VPNStackView.qml [hunk 1]
```diff
@@ -12,8 +12,15 @@ StackView {
...
 
     Component.onCompleted: function(){
-        if(!currentItem && typeof initialItem === "number" ){
+        if(!currentItem && initialItem) {
+            // We don't show anything right now and inital item is set, 
+            // On android if initialItem is anything But a component
+            // it will totaly parse that into garbage values and fail
+            // 
+            // See https://github.com/mozilla-mobile/mozilla-vpn-client/pull/2638
             console.error("Failed to parse initialItem, try Component.OnComplete:push(someURI)");
```
Comment (brizental) [inline nebula/ui/components/VPNStackView.qml:21]:
```suggestion
            console.error("Using the initialItem property does not work on some platforms. Use Component.onCompleted: stackview.push(someURI)");
```
Question, is the documentation we can link to on why this is illegal on Android? Or a bug maybe?

Comment (brizental) [reply nebula/ui/components/VPNStackView.qml:21]: Also, is this only for Android? Is so we can replace "some platforms" with "Android".

File: nebula/ui/components/VPNStackView.qml [hunk 2]
```diff
@@ -12,8 +12,15 @@ StackView {
...
+            VPN.exitForUnrecoverableError("Setting initialItem on a StackView is illegal, dont do that.");
```
Comment (brizental) [inline nebula/ui/components/VPNStackView.qml:22]:
```suggestion
            VPN.exitForUnrecoverableError("Setting initialItem on a StackView is illegal. See previous logs for more information.");
```

**Review by brizental — APPROVED** [review]

File: nebula/ui/components/VPNStackView.qml (hunk 1 above)
Comment (strseb) [reply nebula/ui/components/VPNStackView.qml:21]: >Question, is the documentation we can link to on why this is illegal on Android? Or a bug maybe? 
The pr linked in the comment goes a bit more into detail why, which is the only thing i have found about this issue so far 😅 

>Also, is this only for Android? 
Only android-arm as far as i know.

Comment (brizental) [reply nebula/ui/components/VPNStackView.qml]: Ack sorry, for some reason I thought the linked PR was this one...

**Pull request merged**